view/templates/admin_logs.tpl

```diff
@@ -14,6 +14,7 @@
 
 	<h2>{{$phpheader}}</h2>
 	<div>
+		<p><strong>{{$phplogenabled}}</strong><p>
```

Comment:
you should not use hardcoded stuff like `<strong>`. Rather use `<span class="">` with a class or `<label for="">` depending of what you want to do (have a look in the html docu for explanation). It's definitely better to have the possibility to solve such things with css rather than have hardcoded html tags